back/engines/commercial/matomo/app/services/matomo/client.rb

```diff
@@ -91,6 +91,68 @@ def find_data_subjects(user_id)
       )
     end
 
+    def create_default_custom_dimensions(site_id)
+      return if !site_id || site_id == ''
+
+      custom_dimensions = get_custom_dimensions(site_id)
+      default_custom_dimensions = [
+        { name: 'tenant_name', scope: 'visit' },
```

Comment:
Have you checked that these are the names of the custom dimensions already in the global site?

Reply:
nope, im thinking those were created manually so i would need an api key to get them, or some user credentials to enter the admin panel